### PR TITLE
task 5

### PR DESCRIPTION
**Pull request by CharlesErrington**
fixed badge labels and colors

### CODE REVIEW

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-5)

File: cypress/e2e/project-list.cy.ts [hunk 1]
```diff
@@ -32,7 +32,19 @@ describe("Project List", () => {
...
+                return "Stable";
+              case "warning":
+                return "Warning";
+              case "error":
+                return "Critical";
+              default:
+                return "";
+            }
+          };
+          cy.wrap($el).contains(getStatusText(mockProjects[index].status));
```
Comment (jkettmann) [inline cypress/e2e/project-list.cy.ts:47]: As mentioned in the chat I wasn't sure about a better way. The problem with this test is that you re-create the same logic again. Basically testing your logic with the same logic.

Now looking at it it makes more sense to use a similar approach as with the languages a few lines up.

```
const languageNames = ["React", "Node.js", "Python"];
...
cy.wrap($el).contains(languageNames[index]);
```

File: api/projects.types.ts [hunk 2]
```diff
@@ -5,11 +5,24 @@ export enum ProjectLanguage {
...
 
 export enum ProjectStatus {
-  stable = "stable",
+  stable = "info",
   warning = "warning",
-  critical = "critical",
+  critical = "error",
 }
 
+export const getStatusText = (status: ProjectStatus): string => {
```
Comment (jkettmann) [inline api/projects.types.ts:13]: I wouldn't add this to a `.types` file as this is only supposed to contain types, interfaces, enums and such. 

You could move this function on top of the component like

```
const getStatusText = (status: ProjectStatus) => { 
  ...
}

export function ProjectCard({ project }: ProjectCardProps) {
   ...
```

Comment (jkettmann) [reply api/projects.types.ts:13]: Also, no need to define the return type `string` as this is inferred by TypeScript (this function only returns strings)

**Pull request merged**